docs/ai/tutorials/evaluate-with-reporting.md

```diff
@@ -150,10 +150,10 @@ Run the test using your preferred test workflow, for example, by using the CLI c
 
 ## Generate a report
 
-1. Install the [Microsoft.Extensions.AI.Evaluation.Console](https://www.nuget.org/packages/Microsoft.Extensions.AI.Evaluation.Console) .NET tool by running the following command from a terminal window (update the version as necessary):
+1. Install the [Microsoft.Extensions.AI.Evaluation.Console](https://www.nuget.org/packages/Microsoft.Extensions.AI.Evaluation.Console) .NET tool by running the following command from a terminal window:
 
    ```dotnetcli
-   dotnet tool install --local Microsoft.Extensions.AI.Evaluation.Console --version 9.3.0-preview.1.25164.6
+   dotnet tool install --local Microsoft.Extensions.AI.Evaluation.Console --prerelease
```

Comment:
`9.5.0` GA version of the packages should be live now :) So the `--prerelease` can be removed for all Evaluation packages now except Evaluation.Safety. It is not _required_ though as the commands should still pull the latest stable version of the packages even if `--prerelease` is included.

Reply:
Can we also update the branding (i.e., remove the `(Preview)` suffix) from all docs for all packages except `Evaluation.Safety`? Doing it in a follow-up PR would be fine too.

Reply:
Yes, will track in #46233.